include/bolgenos-ng/page.hpp

```diff
@@ -0,0 +1,33 @@
+#pragma once
+
+#include <bolgenos-ng/stdtypes>
+
+#include "config.h"
+
+
+namespace memory {
+
+
+/// \brief Page in real memory.
+///
+/// Used only for simplifying address arithmetics.
+struct __attribute__((packed)) page_frame_t {
+	uint8_t bytes[PAGE_SIZE]; ///< bytes in real page frame
+};
+static_assert(sizeof(page_frame_t) == PAGE_SIZE, "Wrong size of page_frame_t");
```

Comment:
Check for moving static assertion to .cpp file sine it doesn't needed for using functionality provided by thos header